release/pkg/assets_cni.go

```diff
@@ -37,13 +37,14 @@ func (r *ReleaseConfig) GetCniComponent(spec distrov1alpha1.ReleaseSpec) (*distr
 	for os, arches := range osArchMap {
 		for _, arch := range arches {
 			filename := fmt.Sprintf("cni-plugins-%s-%s-%s.tar.gz", os, arch, gitTag)
-			tarfile := path.Join(r.BuildRepoSource, projectSource, "_output/tar", filename)
+			dirname := fmt.Sprintf("plugins/%s/", gitTag)
+			tarfile := filepath.Join(r.ArtifactDir, dirname, filename)
```

Comment:
nit: we're not using dirname anywhere else
```suggestion
			tarfile := filepath.Join(r.ArtifactDir, "plugins", gitTag, filename)
```